### PR TITLE
RELATED: ONE-3259 Contrast label color

### PR DESCRIPTION
**Pull request by ivan-nejezchleb**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/729/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/288/ : SUCCESS in 1m 31s

File: src/components/visualizations/chart/highcharts/plugins/dataLabelsColors.ts [hunk 1]
```diff
@@ -37,6 +39,27 @@ function setLabelsColor(chart: any) {
...
+function whiteIsNotContrastEnough(color: string) {
+    const { R, G, B } = parseRGBColorCode(color);
+    const colorLuminance = calculateLuminance(R, G, B);
+    const WHITE_LUMINANCE = 1;
+    const CONTRAST_TRESHOLD = 0.4;
+    return (WHITE_LUMINANCE - colorLuminance) < CONTRAST_TRESHOLD;
+}
+
+function setContrastLabelsColor(chart: any) {
+    const points = flatMap(getVisibleSeries(chart), (series: any) => series.points)
```
Comment (pbenes) [inline src/components/visualizations/chart/highcharts/plugins/dataLabelsColors.ts]: we could encap this to a function, let's say `getPointsWithLabel` and reuse from both

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/734/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/293/ : SUCCESS in 2m 26s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/735/ : SUCCESS in 20s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/296/ : FAILURE in 5m 47s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/736/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/297/ : SUCCESS in 1m 26s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/748/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/303/ : SUCCESS in 2m 09s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/758/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/309/ : SUCCESS in 1m 36s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/778/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/328/ : SUCCESS in 1m 37s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/782/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/331/ : SUCCESS in 1m 43s

**Comment by gdgate** [comment]
Build failed (check-teardown pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-stop-zuul-docker/31/ : FAILURE in 1m 19s

**Pull request closed without merging**